public/css/designer.css

```diff
@@ -0,0 +1,215 @@
+html,
+body {
+  height: 100%;
+}
+
+#graph {
+  width: 100%;
+  height: 100%;
+  display: block;
+  z-index: 0;
+}
+
+#graphContainerForBorder {
+  padding: 2px 2px 2px 2px;
+  border: 1px solid #4e555b;
+  height: 71vh;
+}
+
+#variablesDiv {
+  height: 85vh;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+#graphDiv {
+  height: 85vh;
+  padding-left: 15px;
+  padding-right: 15px;
+  display: block;
+}
+
+#graphDivCard {
```

Comment:
For common properties like this one, it's better to create a class that represents it e.g. `.full-height { height: 100%; }`